cube/bmp_8.cpp

```diff
@@ -77,14 +73,14 @@ unsigned char **Bmp8::CreateBmpBuffer(unsigned char init_val) {
 
   buff = (unsigned char **) new unsigned char *[hgt_ * sizeof(*buff)];
   if (!buff) {
```

Comment:
I find this null pointer check after the use of the C++ new operator questionable.
Can it be omitted here?